test/DebuggerCommon/step_in_from_interpreted_function_attach.js.dbg.baseline

```diff
@@ -2,19 +2,19 @@
   {
     "callStack": [
       {
-        "line": 27,
+        "line": 29,

```

Comment:
Since you changed the lines in file you will also need to update full repro baseline at [FullRepro]\unittest\DebuggerCommon\step_in_from_interpreted_function_attach.js.dbg.baseline. And since you need to update full baseline you may not do delete WScript.Platform and have Platform show in dbg baseline